packages/ag-charts-community/src/dom/container.css

```
@@ -45,8 +45,8 @@
     -webkit-user-select: none;
 }
 
-.ag-charts-canvas-container > *,
-.ag-charts-canvas > * {
+.ag-charts-canvas-container > *:not([popover]),
```

I didn't know we had this rule, and it's definitely not right. We need to remove it.

We need to be careful not to change the specificity, however.

```suggestion
.ag-charts-canvas-container > :where(:not([popover])),
```

---

Discussed offline and fixed.